gen-certs.sh

```diff
@@ -18,7 +18,7 @@ certs() {
 }
 
 privatedir() {
-    echo "/etc/pki/private"
+    echo "$(dir)/private"
```

Comment:
Wouldn't it be better to replace `$(dir)` (a function call) by a simple variable?